Possible to release a patch version with fixes to mypy types?

I am trying `v0.2.2` and mypy is complaining about the type when `/` (`__truediv__`) is used to create a path with `--warn-return-any` flag with `mypy`. e.g.:

<details open>

<summary>test.py</summary>

```py
from typing_extensions import reveal_type
from upath import UPath


def foo() -> UPath:
    foo = UPath("s3://bucket/")
    bar = foo / "bar.json"
    reveal_type(bar)
    return bar
```

</details>

```sh
❯ mypy --warn-return-any test.py
test.py:8: note: Revealed type is "Any"
test.py:9: error: Returning Any from function declared to return "UPath"  [no-any-return]
Found 1 error in 1 file (checked 1 source file)

❯ python -m pip show universal_pathlib
Name: universal_pathlib
Version: 0.2.2
Summary: pathlib api extended to use fsspec backends
Home-page: https://github.com/fsspec/universal_pathlib
Author: 
Author-email: 
License: MIT
Location: /Users/rahuliyer/workspace/.venv/lib/python3.10/site-packages
Requires: fsspec
Required-by:
```

I believe this is because `__truediv__` is defined in [`upath/_compat.py`](https://github.com/fsspec/universal_pathlib/blob/v0.2.2/upath/_compat.py#L310) in `v0.2.2` which got fixed with #212.

I was wondering if we can have a patch release which can fix the type checks? If not, I am okay waiting for `v0.3.0` (#223)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ...
 
+## [0.2.3] - 2024-08-23
+### Added
+- upath: add st_birthtime as standard field (#254)
+- upath: added SMBPath and tests (#219)
+- ci: added typesafety checks (#212)
+
+### Fixed
+- upath: fix UPath.rename for absolute paths (#225)
+- upath._flavour: fix path parsing due to change in urllib.parse.SplitResult behavior (#236)
+- upath: fixed typing regressions (#212)
+- upath: update flavour sources (#224, #237, #252)
+- docs: fix link to filesystem spec optional dependencies (#232)
+
 ## [0.2.2] - 2024-03-04
 ### Fixed
 - upath: fixed comparison with pathlib.Path on py<3.12 (#203)
@@ -126,7 +139,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - started a changelog to keep track of significant changes
 
-[Unreleased]: https://github.com/fsspec/universal_pathlib/compare/v0.2.2...HEAD
+[Unreleased]: https://github.com/fsspec/universal_pathlib/compare/v0.2.3...HEAD
+[0.2.3]: https://github.com/fsspec/universal_pathlib/compare/v0.2.2...v0.2.3
 [0.2.2]: https://github.com/fsspec/universal_pathlib/compare/v0.2.1...v0.2.2
 [0.2.1]: https://github.com/fsspec/universal_pathlib/compare/v0.2.0...v0.2.1
 [0.2.0]: https://github.com/fsspec/universal_pathlib/compare/v0.1.4...v0.2.0

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - ci: added typesafety checks (#212)
 
 ### Fixed
+- upath: fix UPath.is_absolute on <3.12 (#256)
 - upath: fix UPath.rename for absolute paths (#225)
 - upath._flavour: fix path parsing due to change in urllib.parse.SplitResult behavior (#236)
 - upath: fixed typing regressions (#212)

